src/v1/components/surfaces/FixedPanel/FixedPanel.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import MUIBox from "@mui/material/Box/Box";
+import Panel from "./index";
+import { FloatingPanelContext } from "../FloatingPanel";
+import { MapIcon } from "../../data-display";
+import { FlexBox } from "../../layout";
+
+const meta: Meta<typeof Panel> = {
+  title: "Surfaces/FixedPanel",
+  component: Panel,
+  tags: ["autodocs"],
+  decorators: (Story) => (
```

Comment:
**Request IF IT DOESN'T EXIST**

**Non-blocking: Can do now or ticket for later**

A story that shows the entire panel system working (without any storybook API confusing things)

<details>
  <summary>rough example</summary>

It'd just be the bare minimum
```tsx
const TARGET_ID = 'uniqueTargetIdForPanel';
const ProviderStory = () =>
    <FloatingPanelContext.Provider
      value={{
        panels: { [TARGET_ID]: { visible: true, minimised: false } },
      }}
    >
      <MUIBox height={500}>
        <Panel
	    count={3}
	    icon={<MapIcon color="primary" fontSize="inherit" />}
	    title="Nodes with geo data"
	    targetId={TARGET_ID}
	    visible>
		<ExampleChildren />
	</MUIBox>
    </FloatingPanelContext.Provider>
```

</details>

Reply:
If it does exist - it probably needs better naming as I can't find it
https://telicent-oss.github.io/telicent-ds/TELFE-660/fixed-panels/?path=/docs/surfaces-floatingpanel-floatingpanel-dockitem--docs&pr=165